src/ui/lib/components/package-section-header.svelte

```diff
@@ -6,13 +6,44 @@
 	import Icon from './icon.svelte';
 	import Tooltip from './tooltip.svelte';
 	import type { IconVariant } from './icon.svelte';
+	import { Paper, type SSX, Typography } from '@ui';
 	export let icon: IconVariant = 'package';
+
+	const ssx: SSX = {
```

Comment:
I can understand what it is and won't block this PR for it, but I have some concerns about how bespoke SSX is for potential newcomers to the project.

![image](https://user-images.githubusercontent.com/3977569/231888094-4313fbe0-1d73-4cab-bf8e-02466e6ce171.png)

Reply:
https://unicorn-ui.netlify.app/scoped-styles

Reply:
https://defenseunicorns.blogin.co/posts/unicorn-ui-why-scopedstyles-200989?s=svelte

Reply:
architected after https://mui.com/system/getting-started/the-sx-prop/

Reply:
Yep yep tracking our own implementation - just wary of bespoke things - even calling it the "sxProp" over ssx may be helpful for people coming to the project.